Fireworks/Geometry/test/test_dumpRecoGeometry.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -x
+function die { echo $1: status $2 ; exit $2; }
+
+# Define the base directory where the geometry files are located
+GEOMETRY_DIR="$CMSSW_RELEASE_BASE/src/Configuration/Geometry/python"
```

Comment:
@smuzaffar I think this will be fine for IBs and full releases but not maybe not for PR and patches ?
Do you have a suggestion to automatically fall back to `$CMSSW_BASE` in those cases ?

Reply:
@mmusich , yes this is not good for PR and patch releases as it will not pack any local changes from $CMSSW_BASE/src/Configuration/Geometry. In many places in cmssw we do something like
```
XDIR="${CMSSW_BASE}/<path>"
if [ ! -e ${XDIR} ] ; then
  XDIR="${CMSSW_RELEASE_BASE}/<path>"
fi
```

We can add a utility script ( either in cmssw itself or in cms-common to be available for all releases) which can return the fullpath for a relative path e.g. `find-cmssw-file relative-path` and it can return either `$CMSSW_BASE/relative-path` or  `$CMSSW_RELEASE_BASE/relative-path`